docs/Remote Control Guide/index.md

```diff
@@ -0,0 +1,185 @@
+
+View **GetSystemCapability** in the [RPC Spec](https://github.com/smartdevicelink/rpc_spec#getsystemcapability)
+
+### GetInteriorVehicleData
+
+GetInteriorVehicleData is used to request information about a specific module. This RPC, provided a module is specified by `moduleType` and `moduleId`, will return the status of the module and if relevant, its' submodules. This RPC can also be used to subscribe to updates of a module's status via the `subscribe` parameter. If this non-mandatory parameter is set to true, the head unit will register `OnInteriorVehicleData` notifications for the requested module. Conversely, if this parameter is set to false, the head unit will unregister `OnInteriorVehicleData` notifications for the requested module.
+
+View **GetInteriorVehicleData** in the [RPC Spec](https://github.com/smartdevicelink/rpc_spec#getinteriorvehicledata) or the [HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/getinteriorvehicledata)
+
+### OnInteriorVehicleData
+
+OnInteriorVehicleData is a notification sent out by the HMI when an update is made to a remote control module. An app can subscribe to these notifications via GetInteriorVehicleData. This RPC will come with a `ModuleData` structure identifying the changed module and containing the control data object with the new state.
+
+View **OnInteriorVehicleData** in the [RPC Spec](https://github.com/smartdevicelink/rpc_spec#oninteriorvehicledata) or the [HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/oninteriorvehicledata)
+
+### SetInteriorVehicleData
+
+SetInteriorVehicleData is used to set the values of a remote control module by passing in a `ModuleData` structure. The `moduleType` and `moduleId` fields are used to identify the targeted module, and the changes in the respective control data object are applied to that module.
+
+View **SetInteriorVehicleData** in the [RPC Spec](https://github.com/smartdevicelink/rpc_spec#setinteriorvehicledata) or the [HMI Documentation](https://smartdevicelink.com/en/guides/hmi/rc/setinteriorvehicledata)
+
+### OnRemoteControlSettings
+
+OnRemoteControlSettings is used to notify SDL when passengers of a vehicle change the remote control settings via the HMI. This includes allowing or disallowing remote control or changing the access mode that will be used for resource allocation.
```

Comment:
```suggestion
OnRemoteControlSettings is used to notify SDL when passengers of a vehicle change the remote control settings via the HMI. This includes allowing or disallowing Remote Control or changing the access mode that will be used for resource allocation.
```

It seems you are using "Remote Control" (capitalized) when referencing the feature itself, so I'm suggesting that change here.